scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-insurance-service.json

```diff
@@ -0,0 +1,182 @@
+        {
+          "name": "OTEL_LOGS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_CONFIGURATOR",
+          "value": "aws_configurator"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER",
+          "value": "xray"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+          "value": "http://localhost:4316/v1/traces"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT",
+          "value": "http://localhost:4316/v1/metrics"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "OTEL_RESOURCE_ATTRIBUTES",
+          "value": "service.name=spring-petclinic-demo-insurance-service"
+        },
+        {
+          "name": "OTEL_METRICS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_DISTRO",
+          "value": "aws_distro"
+        },
+        {
+          "name": "EUREKA_SERVER_URL",
+          "value": "discovery-server-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "INSURANCE_SERVICE_IP",
+          "value": "insurance-service-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "DB_NAME",
+          "value": "postgres"
+        },
+        {
+          "name": "DB_USER",
+          "value": "db-user-name"
+        },
+        {
+          "name": "DB_USER_PASSWORD",
+          "value": "db-user-password"
+        },
+        {
+          "name": "DATABASE_PROFILE",
+          "value": "postgresql"
+        },
+        {
+          "name": "DB_SERVICE_HOST",
+          "value": "db_service_host"
+        },
+        {
+          "name": "DB_SERVICE_PORT",
+          "value": "5432"
+        }
+      ],
+      "command": ["sh", "-c", "python manage.py migrate && python manage.py loaddata initial_data.json && python manage.py runserver 0.0.0.0:8000 --noreload"],
+      "portMappings": [
+        {
+          "containerPort": 8000,
+          "protocol": "tcp"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/pet-clinic-insurance-service",
+          "awslogs-create-group": "true",
+          "awslogs-region": "region-name",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "healthCheck": {
+        "command": [
+          "CMD-SHELL",
+          "curl -f http://localhost:8000/insurances/ || exit 1"
+        ],
+        "interval": 60,
+        "timeout": 10,
+        "retries": 5,
+        "startPeriod": 3
+      },
+      "mountPoints": [
+        {
+          "sourceVolume": "opentelemetry-auto-instrumentation-python",
+          "containerPath": "/otel-auto-instrumentation-python",
+          "readOnly": false
+        }
+      ]
+    },
+    {
+      "name": "init",
+      "image": "adot-python-image",
+      "cpu": 0,
```

Comment:
nit: `cpu` as 0 is the default value, it doesn't have to be explicitly specified.

Reply:
deleted all` "cpu": 0`